### PR TITLE
[GitHub Action] add flow for welcoming first-time-contributors

### PR DESCRIPTION
**Pull request by vishalvivekm**
**Description**

This PR adds action to welcome contributor on opening their first Pull Request.

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by vishalvivekm** [comment]
There's some issue with the body, will fix.

 **Please do not merge**

**Comment by l5io** [comment]
🚀 Preview for commit d1912e6c88ad2df0da98b9cb7e4b4b886ed707fe at: https://66a68db34a43fc6ff90e74f6--layer5.netlify.app

**Comment by sudhanshutech** [comment]
@vishalvivekm is it done?

**Comment by sudhanshutech** [comment]
@vishalvivekm ?

**Comment by vishalvivekm** [comment]
Welcome, @vishalvivekm! Thank you for your first contribution! 🎉 A contributor will be by to give feedback soon. In the meantime, please review the [Newcomers' Guide](https://layer5.io/community/newcomers) and be sure to join the [community Slack](https://slack.layer5.io/).
            <p align="center" width="100%">
            
            </p>
            Be sure to double-check that you have signed your commits. 
            Here are instructions for [making signing an implicit activity while peforming a commit](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin).

File: .github/workflows/first-time-contributor.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            const message = `Welcome, @${{ github.event.pull_request.user.login }}! Thank you for your first contribution! 🎉 A contributor will be by to give feedback soon. In the meantime, please review the [Newcomers' Guide](https://layer5.io/community/newcomers) and be sure to join the [community Slack](https://slack.layer5.io/).
+
+            <p align="center" width="100%">
+            <img src="https://github.com/user-attachments/assets/f3f7d995-fc9a-4aec-8c9d-ed6570fb56ab" width="65%"  />
+            </p>
+
+            Be sure to double-check that you have signed your commits. 
+            Here are instructions for [making signing an implicit activity while peforming a commit](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin).`;
```
Comment (vishalvivekm) [inline .github/workflows/first-time-contributor.yml:22]:
```suggestion
            Here are instructions for [making signing an implicit activity while performing a commit](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin).`;
```

**Comment by l5io** [comment]
🚀 Preview for commit e0861c713ab9cda87dd67fa4ff79cca6d21cb9c9 at: https://66e9820795a9cf12f0e30fd8--layer5.netlify.app

**Comment by sudhanshutech** [comment]
> Welcome, @vishalvivekm! Thank you for your first contribution! 🎉 A contributor will be by to give feedback soon. In the meantime, please review the [Newcomers' Guide](https://layer5.io/community/newcomers) and be sure to join the [community Slack](https://slack.layer5.io/).
> 
> 
> 
> 
> Be sure to double-check that you have signed your commits.
> Here are instructions for [making signing an implicit activity while peforming a commit](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin).

@vishalvivekm its not compatible with dark mode of github.

**Comment by sudhanshutech** [comment]
@vishalvivekm checkiing again.

**Comment by l5io** [comment]
🚀 Preview for commit 770d9489ed2c1678e441c5777fec3d090671da97 at: https://670d211d3d287d30c1b13d62--layer5.netlify.app

**Pull request closed without merging**